harald/arena.py

```diff
@@ -1,2 +1,51 @@
+import abc
+import logging
+from typing import List
+
+import numpy as np
+from numpy.linalg import norm
+
+from harald.agent import Player
+
+logger = logging.getLogger(__name__)
+
+
+class ArenaLayer(abc.ABC):
+    @abc.abstractmethod
+    def force(self, player: Player) -> np.array:
+        """Return force for specific player."""
+
+
+class FrictionLayer(ArenaLayer):
+    def __init__(self, friction_matrix: np.ndarray) -> None:
+        self.friction = friction_matrix
+
+    def force(self, player: Player) -> np.array:
```

Comment:
It would be really nice if you could write some docstrings for these methods.
Docstrings should preferably explain the input arguments and what they mean and what the function returns. In this case, for instance:

```python
def force(self, player: Player) -> np.ndarray:
    """
    Return the friction force which the arena asserts onto the player object.

    :param player: Player for which to calculate the friction force for.
    :return: A numpy vector of length 2 which gives the friction force in (x, y) direction.
    """
```